fiftyone/utils/coco.py

```diff
@@ -1122,6 +1122,9 @@ def from_anno_dict(cls, d, extra_attrs=True):
             a :class:`COCOObject`
         """
         if extra_attrs is True:
+            if "attributes" in d:
```

Comment:
I think the changes here could be reduced to just this one line at the beginning of the method:

```py
# Handles CVAT exported attributes
d.update(d.pop("attributes", {}))
```

As currently written, the attribute handling logic is implemented twice here, which is what initially caught my eye.

Reply:
Good call!